packages/office-ui-fabric-react/src/components/Rating/Rating.tsx

```diff
@@ -120,8 +140,19 @@ export class Rating extends BaseComponent<IRatingProps, IRatingState> {
     );
   }
 
-  private _getIcon(): JSX.Element {
-    return <i className={ css('ms-Icon', this.props.icon || 'ms-Icon--FavoriteStarFill') } />;
+  private _getIcon(starSize?: number): JSX.Element {
+    if (starSize && starSize < 1) {
+      return (
+        <span className={ css('ms-Rating-partialStarContainer', styles.partialStarContainer, 'ms-Icon', this.props.icon || 'ms-Icon--FavoriteStarFill') } >
```

Comment:
We do support overriding icon for all the places - partial as well as full stars